build.sbt

```diff
@@ -27,34 +27,37 @@ scalaVersion := scala212
 crossScalaVersions := supportedScalaVersions
 
 libraryDependencies ++= Seq(
-  "org.apache.spark" %% "spark-sql" % sparkVersion.value % "provided" withSources(),
-  "org.apache.spark" %% "spark-core" % sparkVersion.value % "provided" withSources(),
-  "org.apache.spark" %% "spark-catalyst" % sparkVersion.value % "provided" withSources(),
-  "io.delta" %% "delta-core" % "0.6.1" % "provided" withSources(),
-  "org.apache.iceberg" % "iceberg-spark-runtime" % "0.11.0" % "provided" withSources(),
-
+  "org.apache.spark" %% "spark-sql" % sparkVersion.value % "provided" withSources (),
+  "org.apache.spark" %% "spark-core" % sparkVersion.value % "provided" withSources (),
+  "org.apache.spark" %% "spark-catalyst" % sparkVersion.value % "provided" withSources (),
+  "io.delta" %% "delta-core" % "0.6.1" % "provided" withSources (),
+  "org.apache.iceberg" % "iceberg-spark-runtime" % "0.11.0" % "provided" withSources (),
   // Test dependencies
   "org.scalatest" %% "scalatest" % "3.0.5" % "test",
   "org.mockito" %% "mockito-scala" % "0.4.0" % "test",
   "org.apache.spark" %% "spark-catalyst" % sparkVersion.value % "test" classifier "tests",
   "org.apache.spark" %% "spark-core" % sparkVersion.value % "test" classifier "tests",
-  "org.apache.spark" %% "spark-sql" % sparkVersion.value % "test" classifier "tests"
-)
+  "org.apache.spark" %% "spark-sql" % sparkVersion.value % "test" classifier "tests")
 
 assemblyMergeStrategy in assembly := {
   case PathList("run-tests.py") => MergeStrategy.first
   case x => (assemblyMergeStrategy in assembly).value(x)
 }
 
-scalacOptions ++= Seq(
-  "-target:jvm-1.8"
-)
+scalacOptions ++= Seq("-target:jvm-1.8")
 
 javaOptions += "-Xmx1024m"
 
-/*****************************
- * ScalaStyle configurations *
- *****************************/
+// The following creates target/scala-2.*/src_managed/main/sbt-buildinfo/BuildInfo.scala.
+lazy val root = (project in file("."))
+  .enablePlugins(BuildInfoPlugin)
+  .settings(
+    buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion),
+    buildInfoPackage := "com.microsoft.hyperspace")
```

Comment:
To reviewers, these are the only relevant changes for this PR. Other changes are to conform to the scala style (I ran scalafmt + fixed the comment style)